caching/src/main/java/com/iluwatar/caching/App.java

```diff
@@ -136,4 +137,26 @@ public void useReadThroughAndWriteBehindStrategy() {
     AppManager.find("004");
     System.out.println(AppManager.printCacheContent());
   }
+
+  /**
+   * Cache-Aside
+   */
+  public void useCacheAsideStategy() {
+    System.out.println("# CachingPolicy.ASIDE");
```

Comment:
Is there any reason for not using logging framework like SLF4J or LOG4J Or Common logging?